src/jit/core_templates.expr

```diff
@@ -20,10 +20,6 @@
 
 (template: set (copy $1))
 
-(template: trunc_i8 (ucast $1 int_sz 1))
```

Comment:
These changes don't seem to belong in this PR?

Reply:
These changes are here because I ran the script that sorts the templates so the new listen op would be in the right place.